pkg/rpc/client.go

```diff
@@ -5,13 +5,13 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/CityOfZion/neo-go/pkg/core/entities"
```

Comment:
`goimports`

Reply:
heh yeah, I will go through one more time for sure and check all of that stuff